test/runtests.jl

```diff
@@ -1149,24 +1149,35 @@ end
                         ("'[4,4)'::int4range", Interval{Int32}()),  # Empty interval
-                        ("'[3,7)'::int8range", Interval(Int64(3), Int64(7), true, false)),
-                        ("'(3,7)'::int8range", Interval(Int64(4), Int64(7), true, false)),
-                        ("'[4,4]'::int8range", Interval(Int64(4), Int64(5), true, false)),
+                        ("'[3,7)'::int8range", Interval{Closed, Open}(Int64(3), Int64(7))),
+                        ("'(3,7)'::int8range", Interval{Closed, Open}(Int64(4), Int64(7))),
+                        ("'[4,4]'::int8range", Interval{Closed, Open}(Int64(4), Int64(5))),
                         ("'[4,4)'::int8range", Interval{Int64}()),  # Empty interval
-                        ("'[11.1,22.2)'::numrange", Interval(Decimal(11.1), Decimal(22.2), true, false)),
-                        ("'[2010-01-01 14:30, 2010-01-01 15:30)'::tsrange", Interval(DateTime(2010, 1, 1, 14, 30), DateTime(2010, 1, 1, 15, 30), true, false)),
-                        ("'[2010-01-01 14:30-00, 2010-01-01 15:30-00)'::tstzrange", Interval(ZonedDateTime(2010, 1, 1, 14, 30, tz"UTC"), ZonedDateTime(2010, 1, 1, 15, 30, tz"UTC"), true, false)),
-                        ("'[2004-10-19 10:23:54-02, 2004-10-19 11:23:54-02)'::tstzrange", Interval(ZonedDateTime(2004, 10, 19, 12, 23, 54, tz"UTC"), ZonedDateTime(2004, 10, 19, 13, 23, 54, tz"UTC"), true, false)),
-                        ("'[2004-10-19 10:23:54-02, Infinity)'::tstzrange", Interval(ZonedDateTime(2004, 10, 19, 12, 23, 54, tz"UTC"), ZonedDateTime(typemax(DateTime), tz"UTC"), true, false)),
-                        ("'(-Infinity, Infinity)'::tstzrange", Interval(ZonedDateTime(typemin(DateTime), tz"UTC"), ZonedDateTime(typemax(DateTime), tz"UTC"), false, false)),
-                        ("'[2018/01/01, 2018/02/02)'::daterange", Interval(Date(2018, 1, 1), Date(2018, 2, 2), true, false)),
+                        ("'[11.1,22.2)'::numrange", Interval{Closed, Open}(Decimal(11.1), Decimal(22.2))),
+                        ("'[2010-01-01 14:30, 2010-01-01 15:30)'::tsrange", Interval{Closed, Open}(DateTime(2010, 1, 1, 14, 30), DateTime(2010, 1, 1, 15, 30))),
+                        ("'[2010-01-01 14:30-00, 2010-01-01 15:30-00)'::tstzrange", Interval{Closed, Open}(ZonedDateTime(2010, 1, 1, 14, 30, tz"UTC"), ZonedDateTime(2010, 1, 1, 15, 30, tz"UTC"))),
+                        ("'[2004-10-19 10:23:54-02, 2004-10-19 11:23:54-02)'::tstzrange", Interval{Closed, Open}(ZonedDateTime(2004, 10, 19, 12, 23, 54, tz"UTC"), ZonedDateTime(2004, 10, 19, 13, 23, 54, tz"UTC"))),
+                        ("'[2004-10-19 10:23:54-02, Infinity)'::tstzrange", Interval{Closed, Open}(ZonedDateTime(2004, 10, 19, 12, 23, 54, tz"UTC"), ZonedDateTime(typemax(DateTime), tz"UTC"))),
+                        ("'(-Infinity, Infinity)'::tstzrange", Interval{Open, Open}(ZonedDateTime(typemin(DateTime), tz"UTC"), ZonedDateTime(typemax(DateTime), tz"UTC"))),
```

Comment:
Should use Infinity.jl for these

Reply:
Infinity.jl support is added here: https://github.com/invenia/LibPQ.jl/pull/185